examples/deployments/render-terraform/README.md

```diff
@@ -0,0 +1,118 @@
+# Render.com Deployment
+
+This is an example deployment to Render.com using [terraform](https://www.terraform.io/)
+
+## Requirements
+
+- [Terraform CLI v1.3.4+](https://developer.hashicorp.com/terraform/tutorials/gcp-get-started/install-cli)
+- [Terraform Render provider](https://registry.terraform.io/providers/jackall3n/render/latest/docs)
+
+## Deployment with terraform
+
+### 1. Init your terraform state
+
+```bash
+terraform init
+```
+
+### 3. Deploy your application
+
+```bash
+# Your Render.com API token. IMPORTANT: The API does not work with Free plan.
+export TF_VAR_render_api_token=<render_api_token>
+# Your Render.com user email
+export TF_VAR_render_user_email=<render_user_email>
+#set the chroma release to deploy
+export TF_VAR_chroma_release="0.4.12"
+# the region to deploy to. At the time of writing only oregon and frankfurt are available
+export TF_VAR_region="oregon"
+#enable basic auth for the chroma instance
+export TF_VAR_enable_auth="true"
+#The auth type to use for the chroma instance (token or basic)
+export TF_VAR_auth_type="token"
+terraform apply -auto-approve
+```
+
+### 4. Check your public IP and that Chroma is running
+
+> Note: It might take several minutes for the instance to boot up
```

Comment:
This is why I moved beneggers.com off of Render :/ It's shameful if an infrastructure engineer's personal website takes more than 200ms to load

Reply:
I did another round of updates to the render terraform provider, and now we create the whole think in a single request and from running apply to getting the instance up is roughly a minute, if not less.